### DISCUSSION TITLE
Problems with docker-compose install

### DISCUSSION THREAD

**Discussion by GeorgBNM** [Bottelet/Flarepoint-crm#311] [Q&A]
Hi,

trying to bring DaybydayCRM up with docker-compose ends with

Warning: require(/var/www/html/public/../vendor/autoload.php): failed to open stream: No such file or directory in /var/www/html/public/index.php on line 20

Fatal error: require(): Failed opening required '/var/www/html/public/../vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /var/www/html/public/index.php on line 20

fault!

Any idea, how to get it working?

Thanks,
Georg

**Comment by Elliot-Construct** [comment]
Unfortunately their docker compose is pretty full of errors, it doesn't actually spin up a working solution as of yet.

**Reply by Elliot-Construct** [reply]
I've had to do a lot of digging around to try and get things working. Still not been successful

Errors include:
- Using root as the user in MySQL env settings causes the MySQL container to crash immediately. It gives a message about not using root as the user, to use root use environment variable root_user_password or something similar
- The NGINX configuration needs some work too. - for this I keep getting a welcome to NGINX window.

**Reply by GeorgBNM** [reply]
Hi @Elliot-Construct,

thanks for the information.
I'm glad, that I'm not alone in this case. ;)

**Reply by Bottelet** [reply]
Would you guys mind telling me your docker and docker-compose version? It might not work above Docker version 20.10.17 and docker-compose version 1.25

**Reply by GeorgBNM** [reply]
OK, got me, @Bottelet!
Mine is docker-compose 1.28.5, build 24fb474e.

**Reply by Elliot-Construct** [reply]
Docker version 20.10.14, build a224086349
docker-compose version 1.29.2, build unknown

How come it won't work at these higher versions? What has changed with the updates that makes it incompatible..

Also if it helps I used Portainer on a forked version of the repo in order to try and fiddle around with different elements to see if I could formulate a solution.
